library/core/tests/slice.rs

```diff
@@ -1638,12 +1638,12 @@ fn sort_unstable() {
     use rand::{rngs::StdRng, seq::SliceRandom, Rng, SeedableRng};
 
     // Miri is too slow (but still need to `chain` to make the types match)
-    let lens = if cfg!(miri) { (2..20).chain(0..0) } else { (2..25).chain(500..510) };
+    let lens = if cfg!(miri) { (2..20).chain(0..0) } else { (2..25).chain(200..210) };
     let rounds = if cfg!(miri) { 1 } else { 100 };
 
-    let mut v = [0; 600];
-    let mut tmp = [0; 600];
-    let mut rng = StdRng::from_entropy();
+    let mut v = [0; 300];
+    let mut tmp = [0; 300];
+    let mut rng = StdRng::seed_from_u64(0);
```

Comment:
I guess this doesn't make the test less general, so it's probably ok that this change is not SBF specific.

Reply:
Yep that was the idea